allennlp/commands/__init__.py

```diff
@@ -43,6 +47,7 @@ def main(prog: str = None,
             "serve": Serve(model_overrides),
             "make-vocab": MakeVocab(),
             "elmo": Elmo(),
+            "fine-tune": FineTune(),
```

Comment:
I'm trying it now. 
An early comment: Would be nice to have a way to set the number of epochs.

Reply:
Actually, nvm. The goal of `overrides` is to let people set such extra params. 
That said, would be nice to give a few examples of overrides. Not trivial how to do so (what format? What keys are available? etc)

Reply:
An example struggle in my case: 
I tried all of these: 
```
-o "num_epochs": 5 
-o { "num_epochs": 5 } 
-o "'num_epochs': 5 "
-o "num_epochs: 5 "
```

Until I realized this is the right syntax: 
```
-o "trainer: { num_epochs: 5 }"
```